timelock-impl/src/main/java/com/palantir/atlasdb/timelock/management/DatabaseNamespaceLoader.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.timelock.management;
+
+import java.util.Set;
+
+import com.palantir.paxos.Client;
+
+public class DatabaseNamespaceLoader implements PersistentNamespaceLoader {
+    @Override
+    public Set<Client> getAllPersistedNamespaces() {
+        throw new UnsupportedOperationException("Loading all namespaces from the database is not supported yet!");
```

Comment:
This will eventually be implemented?

Reply:
Yep! I probably should have given a bit more context. Internally this part of the workstream has two phases: 3B (which is ensuring db timelock runs single client namespace integration tests correctly) and 3C (which is giving full compatibility with MNPTLSIT, including the getAllNamespaces tests). Eventually in 3C.3 this will be implemented :)